module/planning/FallingRelaxPlanner/README.md

```diff
@@ -0,0 +1,24 @@
+# FallingRelaxPlanner
+
+## Description
+
+The FallingRelaxPlanner module is used to detect when the robot is actively falling over and run a relax action to remove torque to the servos.
+This is done to prevent damage when the robot impacts the ground, hopefully saving many gear set changes
+
+## Usage
+
+Emit a `message::plan::PlanFallingRelax` as a Task.
+Make sure that it has a lower challenge priority than the GetUp, otherwise it may trigger in the middle of the getup resulting in an infinite loop of falling and half getting up.
+
+## Consumes
+
+- `message::input::Sensors` to determine when the robot is actively falling over
+- `message::plan::PlanFallingRelax` as a Task to activate this module to for the robot falling over
```

Comment:
Hmmm should the Task be something more like "PlanRelax" than "PlanFallingRelax"? Like the walk, we don't say "QuinticWalk" we say "Walk" because it could be any kind of walk. The way we handle falling could be done in different ways (eg some other teams make the robots move their arms out of the way), it's just this provider in the provider group uses the relax method.

Reply:
FallingRelax isn't a specific type of relax though like QuinticWalk, that said maybe PlanRelaxWhenFalling might be a better name? Or maybe just RelaxWhenFalling?

I didn't want to  name it just relax because we don't wanna have tasks that let our robots slack off!

Reply:
Renamed to `RelaxWhenFalling`